### DISCUSSION TITLE
How can I find out what an non-zero exit status caused

### DISCUSSION THREAD

**Discussion by halloleo** [sylikc/pyexiftool#50] [Q&A] [answered]
I start out using PyExiftool again. I use `ExifToolHelper().get_tags` and it works great.

However sometimes I get

> `exiftool.exceptions.ExifToolExecuteError: execute returned a non-zero exit status: 1`

In this case I know _from the outside_ that this is caused by a non-existing image file, but can I see that somehow from Exception what caused the problem?

**Comment by sylikc** [comment, marked as answer]
Hi, I have [a new FAQ](https://github.com/sylikc/pyexiftool/pull/48/commits/0984870fbc6dd0822cc9239a8a69274695a052bb#diff-399cec32862278b1cf76d3019a66416c9c5c00e55a067554a69ebebba7684ef3R106) (that's not yet published) it's tied to the [new binary PR changes](https://github.com/sylikc/pyexiftool/pull/48) ... but basically it has some debug instructions:

```python
import logging
import exiftool

logging.basicConfig(level=logging.DEBUG)
with exiftool.ExifToolHelper(logger=logging.getLogger(__name__)) as et:
        et.get_tags( ... )
```

**Reply by halloleo** [reply]
Thanks for this @sylikc! This is certainly an improvement to the current release.
